src/call.ts

```diff
@@ -120,9 +120,7 @@ export class OngoingCallPromise extends OngoingCall {
       if (err) {
         if (err.metadata) {
           const decoder = new GoogleErrorDecoder();
-          err.statusDetails = decoder.decodeRpcStatusDetails(
-            err.metadata.get('grpc-status-details-bin') as Buffer[]
-          );
+          rejectCallback(decoder.decodeMetadata(err));
```

Comment:
rejectCallback will be called twice (lines 123 and 124) which is normally unexpected by the user.

Reply:
Thanks for capturing this. Made change.